libs/payments/capability/src/lib/capability.manager.spec.ts

```diff
@@ -36,17 +36,17 @@ describe('CapabilityManager', () => {
         ],
       }).compile();
 
-      manager = module.get<CapabilityManager>(CapabilityManager);
+      capabilityManager = module.get<CapabilityManager>(CapabilityManager);
     });
 
     it('should be defined', async () => {
-      expect(manager).toBeDefined();
-      expect(manager).toBeInstanceOf(CapabilityManager);
+      expect(capabilityManager).toBeDefined();
+      expect(capabilityManager).toBeInstanceOf(CapabilityManager);
     });
 
     it('should return empty results', async () => {
-      mockResult.getServices = jest.fn().mockReturnValueOnce(undefined);
-      const result = await manager.getClients();
+      mockServicesResult.getServices = jest.fn().mockReturnValueOnce(undefined);
+      const result = await capabilityManager.getClients();
       expect(result.length).toBe(0);
```

Comment:
nit: and only because I believe I mentioned it in a previous PR thread. You can use Jest's [`.toHaveLength()`](https://jestjs.io/docs/expect#tohavelengthnumber) instead of `.toBe()` here.
```suggestion
      expect(result).toHaveLength(0);
```

Although, as always, might be well outside the scope of this general PR and best left as a backlog item.
```sh
git grep -Pin "\.length\)\.toBe\(" libs apps | wc -l # 31
git grep -Pin "\)\.toHaveLength\(" libs apps | wc -l # 7
```

Reply:
Great catch! Not sure if I wrote the original code but that's good to know as I have be sometimes used toBe() and this is much better. Thank you, Peter!! 💟